### PR TITLE
added instance method for add

### LINKED ISSUE #13041: add

### PR DESCRIPTION
**Pull request by AkshayC2001**
Close #13041

### CODE REVIEW

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Apologies for the delay in reviewing! I've left the comments below, please request my review again when you're done with the changes! Thanks a lot!

File: ivy/functional/frontends/tensorflow/tensor.py [hunk 1]
```diff
@@ -201,6 +201,8 @@ def __rxor__(self, x, name="rxor"):
 
     def __sub__(self, y, name="sub"):
         return tf_frontend.math.subtract(self._ivy_array, y, name=name)
+    def __add__(self, y, name="add"):
```
[the remark this answers is not in the document]
Comment (AkshayC2001) [reply ivy/functional/frontends/tensorflow/tensor.py]: hello @juliagsy is this correct?

**Review by juliagsy — COMMENTED** [review]
Hey! Apologies for this but I just realised, the `add` instance method actually doesn't exist in `tensorflow` itself, so we should not be including this in the the ivy frontend too. This method is not present in the ToDo list as well: https://github.com/unifyai/ivy/issues/3616
The `__add__` method is ticked in the todo list so that shouldn't be chosen as it is completed, could you please select a different function/method to work on as this is not a valid one, thanks a lot!

**Comment by AkshayC2001** [comment]
I'll do thank you

**Pull request closed without merging**